content/news/XXXX-XX-XX-Midsummer-Mixxx.md

```diff
@@ -0,0 +1,65 @@
+title: Midsummer Mixxx
+authors: Evelynne Veys
+tags: summer, info, news, update, harmonic, key, stem
+comments: yes
+status: draft
+
+#### Let the Mixxx shine
+
+Summertime, partytime. In the middle of the festival season you are maybe playing a sunny vibes gig somewhere.
+Or you are relaxing on the beach or at the pool with a cocktail in the hand checking the long list of music you had to discover.
+Both ways will give you inspiration and energy.
+Who thinks Mixxx-devs are laying in the sun can't be more wrong, we're having a crowded github-festival ourselves.
+
+#### Let the Mixxx in
+
+A lot of new contributors found their way to our mapping parties on the discourse forum, our coding raves at github, 
+and the development zulip dancehall. You are all welcome and we'd like to thank you for all kind of input.
+With your feedback on the 2.5 beta version the coding team is on track for gold, the bugs and errors teams are already defeated.
+
+#### I'm going on a holiday and ..
+
+While your enjoying a well deserved vacation the Mixxx team is preparing new features.
+Not all new features are already for bundling in an official release but you can find everything about them in the partie venues. 
+
+The new visual aids for key and bpm will be astonishing, it needed quite some adaptations but the project is in pole position for gold.
+
+The color integration will give you intuitive directions of matching tracks for harmonic mixing, also for people who suffer off color-blindness.
+Soon you wil be able to discover a 'heat column' in the library which will give you hints on tracks that are compatible with the current playing track,
+based on the Key and BPM. And BPM coloring will be added to show tracks with a BPM close to the current tempo.
+This will be our first step in Intelligent Assistance, it can inspire you to make surprising Mixxx-es with tracks that slipped your mind.
+And these features will later be used later for some QOL features, like an improved suggestions algorithm, search ranking based on similarity,
+and adding key color ticks on the rate sliders.
+Fill the floors!
+
+Our marathon stem project is fantastic, a tip of the veil:
+Playing with stems changes the whole idea of DJ-ing. If stems are new for you I would like to invite you to do some research. 
```

Comment:
maybe we can link out to some more information about stems? or a youtube demo?

Reply:
at the moment there are a lot of demo-videos created by commercial DJ-Software creators that explain very well what stems are, and how they can be used. Last week there was a release of T4 with nice video's, I imagine a lot of people have seen them. It feels a little odd to link to T not?